ci-operator/step-registry/hypershift/mce/aws/create/hypershift-mce-aws-create-chain.yaml

```diff
@@ -19,6 +19,9 @@ chain:
         - name: ENABLE_ICSP
           default: "false"
           documentation: "if true, add image content sources config(path=${SHARED_DIR}/mgmt_icsp.yaml)"
+        - name: HYPERSHIFT_INSTANCE_TYPE
+          default: "m5.xlarge"
+          documentation: "Instance type for the cluster nodes"
```

Comment:
Is it possible to use a small instance to save the cost ?

Reply:
The default value for HYPERSHIFT_INSTANCE_TYPE is m5.xlarge. If you want to change the default value, you need to override HYPERSHIFT_INSTANCE_TYPE. As far as I know, we should not be able to use smaller instances, as this would result in many failed cases.